docs/Other SDL Features/Setting the Built-in Nav System Destination/index.md

```diff
@@ -1,17 +1,20 @@
-# Setting the Navigation Destination
-Setting a Navigation Destination allows you to send a GPS location, prompting the user to navigate to that location using their embedded navigation. When using the `SendLocation` RPC, you will not receive a callback about how the user interacted with this location, only if it was successfully sent to Core and received. It will be handled by Core from that point on using the embedded navigation system.
+# Setting the Built-In Navigation Destination
+The `SendLocation` RPC gives you the ability to send a GPS location to the embedded navigation app on the head unit. When the request is sent the user will be prompted to navigate to that location using the embedded navigation app. 
```

Comment:
See above.

Reply:
I added a note with a link App Services. Since the section is named **Setting the Built-In Navigation Destination**, I don't think we should add more information about App Services unless we want to reconfigure the section.